client/my-sites/email/email-management/list/email-management-list-item.jsx

```diff
@@ -0,0 +1,358 @@
+			);
+		}
+
+		return null;
+	}
+
+	getAddEmailHandler( domain ) {
+		if ( hasTitanMailWithUs( domain ) ) {
+			return this.onAddTitanMailbox;
+		}
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return this.onAddGoogleUser;
+		}
+
+		if ( hasEmailForwards( domain ) ) {
+			return this.onAddEmailForward;
+		}
+
+		return null;
+	}
+
+	getEmailProductName( domain ) {
+		if ( hasTitanMailWithUs( domain ) ) {
+			return getTitanProductName();
+		}
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return domain.productName ?? getGoogleMailServiceFamily();
+		}
+
+		if ( hasEmailForwards( domain ) ) {
+			return this.props.translate( 'Email Forwarding' );
+		}
+
+		return null;
+	}
+
+	getMailboxCount( domain ) {
+		if ( hasTitanMailWithUs( domain ) ) {
+			return getMaxTitanMailboxCount( domain );
+		}
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return getGSuiteMailboxCount( domain );
+		}
+
+		if ( hasEmailForwards( domain ) ) {
+			return getEmailForwardsCount( domain );
+		}
+
+		return 0;
+	}
+
+	renderFoldableContent() {
+		const { domain, translate } = this.props;
+
+		const addEmailHandler = this.getAddEmailHandler( domain );
+		const wrappedAddEmailHandler = ( event ) => {
+			event.preventDefault();
+			addEmailHandler();
+		};
+
+		return (
+			<React.Fragment>
+				{ this.renderEmails() }
+				{ this.canUserAddEmailAccounts( domain ) && (
+					<VerticalNavItem path="#" onClick={ wrappedAddEmailHandler }>
+						{ translate( 'Add new mailbox' ) }
+					</VerticalNavItem>
+				) }
+			</React.Fragment>
+		);
+	}
+
+	renderEmails() {
+		const { domain, emails, hasGSuiteUsersLoaded } = this.props;
+
+		if ( hasGSuiteWithUs( domain ) && ! hasGSuiteUsersLoaded ) {
+			return (
+				<CompactCard className="email-management-list-item__foldable-content is-placeholder" />
+			);
+		}
+
+		if ( emails.length === 0 ) {
+			return null;
+		}
+
+		return (
+			<React.Fragment>
+				{ emails.map( ( email ) => this.renderEmailAddress( email ) ) }
+			</React.Fragment>
+		);
+	}
+
+	renderEmailAddress( emailDetails ) {
+		const { translate } = this.props;
+
+		return (
+			<CompactCard
+				className="email-management-list-item__foldable-content-email-item"
+				key={ `email-address-${ emailDetails.email }` }
+			>
+				<MaterialIcon icon="email" />
+				<span className="email-management-list-item__foldable-content-email-address">
+					{ emailDetails.email }
+				</span>
+				{ emailDetails.isAdmin && <Badge type="info">{ translate( 'Admin' ) }</Badge> }
+			</CompactCard>
+		);
+	}
+
+	render() {
+		const { domain, translate } = this.props;
+
+		const addEmailButton = null && this.canUserAddEmailAccounts( domain ) && (
+			<Button onClick={ this.getAddEmailHandler( domain ) } compact primary>
+				{ translate( 'Add email user' ) }
+			</Button>
+		);
+
+		const numberOfMailboxes = this.getMailboxCount( domain );
+		const mailboxCountText = numberOfMailboxes
+			? '- ' +
+			  translate( '%(numberOfMailboxes)d mailbox', '%(numberOfMailboxes)d mailboxes', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **8** times:
`translate( 'You have %(numberOfMailboxes)d unused mailbox', 'You have %(numberOfMailboxes)d unused mailboxes' )` *ES Score: 7*